prime-dotnet-webapi/Controllers/EnrolleesController.cs

```diff
@@ -926,5 +926,142 @@ public async Task<ActionResult> GetPlrData(int enrolleeId)
             var result = await _plrProviderService.GetPlrDataByCollegeIdsAsync(collegeIds);
             return Ok(result);
         }
+
+        // POST: api/Enrollees/5/enrollee-absence
+        /// <summary>
+        /// Creates a new enrollee absence.
+        /// </summary>
+        /// <param name="enrolleeId"></param>
+        /// <param name="createModel"></param>
+        [HttpPost("{enrolleeId}/enrollee-absence", Name = nameof(CreateEnrolleeAbsence))]
+        [ProducesResponseType(StatusCodes.Status401Unauthorized)]
+        [ProducesResponseType(StatusCodes.Status403Forbidden)]
+        [ProducesResponseType(typeof(ApiMessageResponse), StatusCodes.Status404NotFound)]
+        [ProducesResponseType(StatusCodes.Status204NoContent)]
+        public async Task<ActionResult> CreateEnrolleeAbsence(int enrolleeId, EnrolleeAbsenceViewModel createModel)
+        {
+            var record = await _enrolleeService.GetPermissionsRecordAsync(enrolleeId);
+            if (record == null)
+            {
+                return NotFound($"Enrollee not found with id {enrolleeId}");
+            }
+            if (!record.MatchesUserIdOf(User))
```

Comment:
if the intention is to exclude adjudicators from calling this endpoint, we should add `[Authorize(Roles  = Roles.PrimeEnrollee)] to this method.